topics/metagenomics/tutorials/general-tutorial/tutorial.md

```diff
@@ -82,7 +82,7 @@ has been sequenced using 454 GS FLX Titanium. For the tutorial, the original fas
 
 > ### {% icon hands_on %} Hands-on: Data upload
 >
-> 1. Import from [Zenodo](https://zenodo.org/record/815875) or from the data library (in "Analyses of metagenomics data") the files
+> 1. Import the following files from [Zenodo](https://zenodo.org/record/815875), or from the shared data library (in `GTN - Material -> Metagenomics -> Analyses of metagenomics data - The global picture`):
```

Comment:
we could even use Jekyll variables here (topic name, tutorial name) so that if we change them we do not need to update this line.
we could then add that to the planemo template
What do you think @shiltemann?

Reply:
very good point :+1:, will update